js/bootstrap-slider.js

```diff
@@ -1024,13 +1019,19 @@
 						}
 
 						if (this.tickLabels[i]) {
-							this.tickLabels[i].style[styleSize] = labelSize + 'px';
-
+							this.tickLabels[i].style[styleSize] = labelSize + '%';
+
+							this.tickLabels[i].style.position = 'absolute';
+							if (this.options.orientation === 'horizontal') {
+				                this.tickLabels[i].style['transform'] = 'translate(-50%, 0%)';
```

Comment:
One question why are you doing the translate in JS and not in CSS? (as i have done in my fork)
```
.slider.slider-vertical .slider-tick-label {
    padding-left: 4px;
    -moz-transform: translateX(50%) translateY(-50%);
    -webkit-transform: translateX(50%) translateY(-50%);
    transform: translateX(50%) translateY(-50%);
}

.slider.slider-horizontal .slider-tick-label {
    -moz-transform: translateX(-50%) translateY(0%);
    -webkit-transform: translateX(-50%) translateY(0%);
    transform: translateX(-50%) translateY(0%);
}
````